types/knockout/index.d.ts

```diff
@@ -443,11 +439,18 @@ interface KnockoutStatic {
     contextFor(node: any): any;
     isSubscribable(instance: any): instance is KnockoutSubscribable<any>;
     toJSON(viewModel: any, replacer?: Function, space?: any): string;
-    toJS<T>(viewModel: KnockoutObservableArray<T>|KnockoutObservableType<T>[]|KnockoutObservableArray<KnockoutObservableType<T>>|T[]): T[];
-    toJS<T>(viewModel: KnockoutObservable<T>|KnockoutObservableType<T>|KnockoutObservable<KnockoutObservableType<T>>|T): T;
-    isObservable<T>(instance: KnockoutObservable<T>|T): instance is KnockoutObservable<T>;
-    isWriteableObservable<T>(instance: KnockoutObservable<T>|T): instance is KnockoutObservable<T>;
-    isComputed<T>(instance: KnockoutObservable<T>|T): instance is KnockoutComputed<T>;
+  
+    toJS(viewModel: any): any;
+
+    isObservable(instance: any): instance is KnockoutObservable<any>;
+    isObservable<T>(instance: KnockoutObservable<T>): instance is KnockoutObservable<T>;
```

Comment:
Input of type T is still valid for this function.

Reply:
You might want to consider adding:

     isObservable<T>(instance: KnockoutObservable<T>): false;

Or changing it to:

     isObservable<T>(instance: KnockoutObservable<T> | T): instance is KnockoutObservable<T>;